Hack so that lldb can find its custom readline.so, because we move it
after install.

This is based on Fedora package.

diff --git a/source/Interpreter/embedded_interpreter.py b/source/Interpreter/embedded_interpreter.py
index a487592..f759aef 100644
--- a/source/Interpreter/embedded_interpreter.py
+++ b/source/Interpreter/embedded_interpreter.py
@@ -1,4 +1,5 @@
 import sys
+sys.path.insert(1, '%{python_sitearch}/lldb')
 
 if sys.version_info[0] < 3:
     import __builtin__ as builtins
